nemo/collections/llm/gpt/data/fine_tuning.py

```diff
@@ -54,10 +54,7 @@ def __init__(
         self.seq_length = seq_length
         self.seed = seed
         self.dataset_root = Path(dataset_root)
-
-        from nemo.collections.nlp.modules.common.tokenizer_utils import get_nmt_tokenizer
-
-        self.tokenizer = tokenizer or get_nmt_tokenizer("megatron", "GPT2BPETokenizer")
+        self.tokenizer = tokenizer
```

Comment:
What's the reason for this change?

Reply:
I don't think it made sense to put a default tokenizer here. Can you tell me your use case? In practice, it was downloading a megatron-345m model every time, so I wanted to prevent that

Reply:
Makes sense, was just wondering why this was added to this PR. The tokenizer is no longer optional now, right? Should we change the type hint too?

Reply:
A user doesn't need to specify this, because the llm.finetune API will link the tokenizer from the imported model